docs/config.rst

```diff
@@ -53,7 +53,7 @@ In order to set the environment and debug mode reliably, Flask uses
 environment variables.
 
 The environment is used to indicate to Flask, extensions, and other
-programs, like Sentry, what context Flask is running in. It is
+programs, like Sentry, what context they are running in. It is
```

Comment:
This is correct as-is. The paragraph is talking specifically about Flask's environment (`app.env`, `FLASK_ENV`).